package.json

```diff
@@ -11,6 +11,7 @@
     "start-otf": "FEATURE_ONBOARDING=true FEATURE_PASSCODE=true npm start -- --reset-cache",
     "start-experiments": "FEATURE_DEV_EXPERIMENTS=true npm start -- --reset-cache",
     "start-icca": "RN_SRC_EXT=icca react-native start",
+    "start-passcode": "FEATURE_PASSCODE=true npm start -- --reset-cache",
```

Comment:
I don't see where this FEATURE_PASSCODE is read / used, or is that coming in a later PR?

Reply:
https://github.com/digidem/mapeo-mobile/blob/33df94085dff0537c339cdca67cb8b02aa7a2930/src/frontend/screens/Settings/Settings.tsx#L177